source/h1_connection.c

```diff
@@ -731,6 +738,14 @@ static struct aws_h1_stream *s_update_outgoing_stream_ptr(struct aws_h1_connecti
     /* If current stream is done sending data... */
     if (current && !aws_h1_encoder_is_message_in_progress(&connection->thread_data.encoder)) {
         current->is_outgoing_message_done = true;
+        if (current->base.on_metrics) {
+            AWS_ASSERT(current->base.metrics.send_end_timestamp_ns == 0);
```

Comment:
if im reading docs correctly, aws_high_res_clock_get_ticks cannot be directly translated into system time. should we be more careful when exposing this value to user and warn them that timestamps are only good for relative operations?

Reply:
yeah, switched to aws_sys_clock_get_ticks